Gemfile

```diff
@@ -7,8 +7,8 @@ group :test do
   gem 'voxpupuli-test', '~> 7.0',   :require => false
   gem 'coveralls',                  :require => false
   gem 'simplecov-console',          :require => false
-  gem 'webrick',                    :require => false if RUBY_VERSION >= '3.2'
   gem 'puppet_metadata', '~> 3.5',  :require => false
+  gem 'webrick',                    :require => false if RUBY_VERSION >= '3.2'
```

Comment:
I'm wondering why we only pull it in on Ruby 3.2, why not Ruby 2.7? is the gem even used?

Reply:
i have no idea. thats something treydock worked on i think